src/ray/plasma/store_exec.cc

```diff
@@ -50,11 +60,20 @@ int main(int argc, char *argv[]) {
   }
 
   if (!keep_idle) {
+    ArrowLog::InstallFailureSignalHandler();
```

Comment:
Does this interfere with our ray log signal handler? We should modify plasma to use ray log

Reply:
no, because this is a standalone plasma process. and I will remove arrow logs in another PR.